docs/aws_function.rst

```diff
@@ -0,0 +1,7 @@
+aws\_function module
```

Comment:
personally, i dont think we need this in the docs. cuz it is more about our deployment than the applications/library code.

since we have not been meeting, I have not been able to clarify. We are mixing our deployment (AWS) with the application code. Usually, deployment code and application code are kept separate. Though for our purposes, I think that would over complicate things.